submissions/israelOladosu/DomManipulation/index.js

```diff
@@ -0,0 +1,58 @@
+let submitForm = document.getElementById("validateForm"),
+  qty = document.getElementById("qty"),
+  shipping_cost = document.getElementById("shipping_cost"),
+  total_cost = document.getElementById("total_cost"),
+  price = document.getElementById("price");
+
+const RegEx = /^[0-9]$/;
+
+document.addEventListener("DOMContentLoaded", function () {
+  if (+price.lastChild.innerText < 1000) {
```

Comment:
Why is there a `+` sign in your variable names

Reply:
same as `parseInt()`, its to convert my price from string to number